### ISSUE TITLE
globals.mzn conflicts with chaffed

### ISSUE THREAD

**Issue by yurivict** [MiniZinc/libminizinc#472]
This example:
```
include "globals.mzn";

var bool: BOOL01;
var bool: BOOL03;
var bool: BOOL05;
var bool: BOOL07;
var 0..1: INT02;
var 0..1: INT04;
var 0..1: INT06;
var 0..1: INT08;
array [1..2] of var 0..2: s;
constraint int_eq_reif(s[1], 0, BOOL01);
constraint int_eq_reif(s[2], 0, BOOL03);
constraint int_eq_reif(s[1], 1, BOOL05);
constraint int_eq_reif(s[2], 1, BOOL07);
constraint bool2int(BOOL01, INT02);
constraint bool2int(BOOL03, INT04);
constraint bool2int(BOOL05, INT06);
constraint bool2int(BOOL07, INT08);
constraint int_lin_eq([-1, -1, 1], [INT02, INT04, s[1]], 0);
constraint int_lin_eq([-1, -1, 1], [INT06, INT08, s[2]], 0);
solve satisfy;
```
fails:
```
Running untitled_model.mzn
/usr/local/share/minizinc/std/nvalue_fn.mzn:7.57-67:
MiniZinc: type error: no function or predicate with this signature found: `nvalue(var int,array[$_] of var int)'
Cannot use the following functions or predicates with the same identifier:
function var int : nvalue(array[$_] of var int: x);
    (requires 1 argument, but 2 given)
predicate nvalue(var int: n,array[int] of var int: x);
    (argument 2 expects type array[int] of var int, but type array[$_] of var int given)

Process finished with non-zero exit code 1
Finished in 243msec
```

```globals.mzn``` conflicts with chaffed. (I know that ```globals.mzn``` isn't needed for this code, but it shouldn't conflict anyway.)

**Comment by cyderize** [comment]
Are you sure you don't have an old version of Chuffed installed? Only the `develop` branch of Chuffed is compatible with the current MiniZinc library - and this is what is included in the IDE bundle.

**Comment by yurivict** [comment]
I use the last released version chuffed- 0.10.4.
It works in general, except for this problem.

**Comment by Dekker1** [comment]
Like @cyderize said, only the `develop` version of Chuffed is fully compatible with the current version of MiniZinc.

This is an issue in the MiniZinc library in the Chuffed library, but has been resolved on the `develop` branch.
